src/serlio/modifiers/PRTModifierAction.cpp

```diff
@@ -168,27 +169,27 @@ std::list<MObject> getNodeAttributesCorrespondingToCGA(const MFnDependencyNode&
 
 const RuleAttribute RULE_NOT_FOUND{};
 
+const RuleAttribute reverseLookupAttribute(const std::wstring& mayaFullAttrName, const RuleAttributes& ruleAttributes) {
+	auto it = std::find_if(ruleAttributes.begin(), ruleAttributes.end(),
+	                       [&mayaFullAttrName](const auto& ra) { return (ra.mayaFullName == mayaFullAttrName); });
+	if (it != ruleAttributes.end())
+		return *it;
+	return RULE_NOT_FOUND;

```

Comment:
as `RULE_NOT_FOUND` is not used anywhere, you could also just write `{}`

Reply:
removed.